### PR TITLE
fix undeclared variables or wrong variable names (/src) part 3

### PR DESCRIPTION
**Pull request by rabuzarus**
Some variables were not declared. Some variables were not available. And some variables were did have a typo.
This PR fixes this issue for the files in the /src directory.

There are two issues left in /src/Model/Item.php (see `@todo`):
Can we move the declaration of `$uid` (L305) up?

Note:
By adding missing variables the code should be fixed but it could have an impact on how the code will work. Keep this in mind when reviewing the PR.

### CODE REVIEW

File: src/Model/Item.php [hunk 1]
```diff
@@ -263,6 +263,7 @@ public static function insert($arr, $force_parent = false, $notify = false, $don
 		}
 
 		$arr['guid'] = notags(trim(defaults($arr, 'guid', get_guid(32, $guid_prefix))));
+		/// @todo Declare $uid variable - Rabuzarus - 2018-02-12.
```
Comment (MrPetovan) [inline src/Model/Item.php]: Yep, it should be `$arr['uid']`.

File: src/Model/Item.php [hunk 2]
```diff
@@ -1294,6 +1295,7 @@ private static function tagDeliver($uid, $item_id)
 			return;
 		}
 
+		/// @todo Declare $r or lets remove it - Rabuzarus - 2018-02-12.
```
Comment (MrPetovan) [inline src/Model/Item.php]: Let's remove it, the hook isn't called anyway.

**Comment by MrPetovan** [comment]
Related to #4176

File: src/Model/Item.php (hunk 1 above)
Comment (annando) [reply src/Model/Item.php]: Or line 306 should be moved some lines higher.

**Comment by MrPetovan** [comment]
You missed these previously undefined variable:
- https://github.com/friendica/friendica/blob/develop/src/Content/Text/BBCode.php#L1138
- https://github.com/friendica/friendica/blob/develop/src/Core/Worker.php#L1021

**Comment by annando** [comment]
Args ... the thing in the worker is a huge bug that I just created these days.

**Pull request merged**